integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOpUpgradeFmwDomainInPV.java

```diff
@@ -0,0 +1,684 @@
+// Copyright (c) 2020, 2021, Oracle Corporation and/or its affiliates.
```

Comment:
"Oracle Corporation" -> "Oracle". Legal changed the required licensing text.

Reply:
fixed